README.md

```diff
@@ -2,8 +2,15 @@
 Status](https://travis-ci.org/18F/fec-eregs.svg?branch=master)](https://travis-ci.org/18F/fec-eregs)
 
 # FEC's eRegs
+
+## Code Status:
+
+### Quantified Code
 [![Code Issues](https://www.quantifiedcode.com/api/v1/project/e2ee92b5c3db486f89d47371c4d89a2f/badge.svg)](https://www.quantifiedcode.com/app/project/e2ee92b5c3db486f89d47371c4d89a2f)
 
+### Gemnasium
+[![Dependency Status](https://gemnasium.com/badges/github.com/18F/fec-eregs.svg)](https://gemnasium.com/github.com/18F/fec-eregs)
+
```

Comment:
No need to put a header on each of these, I would just inline all of them like [atf-eregs](https://github.com/18F/atf-eregs#status) does.